README.md

```diff
@@ -10,6 +10,12 @@ This documentation includes both guidelines, examples and source code docs.
 
 The documentation is accessible at https://cumulus-nasa.github.io/
 
+## Cumulus tasks list
```

Comment:
I don't think this is really necessary in the README since we have it in the documentation